.github/workflows/run-tests.yml

```diff
@@ -49,9 +49,9 @@ jobs:
           distribution: temurin
           cache: maven
       - name: Test core
-        run: mvn test --threads 2C --batch-mode -no-transfer-progress --update-snapshots -f ./dhis-2/pom.xml -pl -dhis-web-embedded-jetty
+        run: mvn test --threads 2C --batch-mode -no-transfer-progress -f ./dhis-2/pom.xml -pl -dhis-web-embedded-jetty
```

Comment:
maybe `-` missing in `-no-transfer-progress` => `--no-transfer-progress`

Reply:
Done